openh264/openh264_make_archive

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+#
+# Archive last build of openh264 to :POST and make a zip file from its contents
+#
+# Contact openSUSE Release Team for more information or contacts
+# https://en.opensuse.org/openSUSE:Release_team
+#
+# More details about OpenH264 in openSUSE at https://en.opensuse.org/OpenH264"
+
```

Comment:
I am not sure how much noise it should produce but at least `set -e` would be nice to see if something failed. `-x` is probably a bit too much.

Reply:
-x is what I used while putting it together :-)

The -e already helps in case that user doesn't have permissions to see the project

```
# with set -e
lkocman@localhost:~/Workspace/opensuse/openSUSE-release-tools/openh264> ./openh264_make_archive 

This tool will make zip archive with binaries from openSUSE:Factory:openh264:POST
This zip archive can be then sent over to Cisco for extraction at ciscobinary.openh264.org.

Please make sure that you've made snapshot of data (release_to_post.sh) prior proceeding.

Press Enter to proceed or ctrl+c to cancel.

/tmp/cisco-binary-opensuse-L2HO/checkout ~/Workspace/opensuse/openSUSE-release-tools/openh264
Server returned an error: HTTP Error 404: Not Found
Project not found: openSUSE:Factory:openh264:POST

# without set -e
lkocman@localhost:~/Workspace/opensuse/openSUSE-release-tools/openh264> vim openh264_make_archive 
lkocman@localhost:~/Workspace/opensuse/openSUSE-release-tools/openh264> ./openh264_make_archive 

This tool will make zip archive with binaries from openSUSE:Factory:openh264:POST
This zip archive can be then sent over to Cisco for extraction at ciscobinary.openh264.org.

Please make sure that you've made snapshot of data (release_to_post.sh) prior proceeding.

Press Enter to proceed or ctrl+c to cancel.

/tmp/cisco-binary-opensuse-jljM/checkout ~/Workspace/opensuse/openSUSE-release-tools/openh264
Server returned an error: HTTP Error 404: Not Found
Project not found: openSUSE:Factory:openh264:POST
./openh264_make_archive: line 33: pushd: openSUSE:Factory:openh264:POST: No such file or directory
./openh264_make_archive: line 37: pushd: *: No such file or directory
No project specified
~/Workspace/opensuse/openSUSE-release-tools/openh264
/tmp/cisco-binary-opensuse-jljM ~/Workspace/opensuse/openSUSE-release-tools/openh264
	zip warning: zip file empty


Please send email with /tmp/cisco-binary-opensuse-jljM/cisco-openh264-rpms-opensuse-2023-01-05.zip to Cisco to get it extracted at ciscobinary.openh264.org.

Execute publish_repodata_openh264.sh !ONLY! once you get confirmation that data was extracted at ciscobinary.openh264.org.

```

Reply:
implemented